site/learn/expert-advice/dev-tools/uuid-generator.html

```diff
@@ -30,4 +31,27 @@
   </form>
 </div>
 
-<script type="text/javascript" src="/assets/js/UUIDGenerator-0.1.0.js" defer></script>
\ No newline at end of file
+<h3>UUID Frequently Asked Questions</h3>
+<h4>What is a UUID?</h4>
+<p>A UUID (Universally Unique Identifier) is a string of characters that is used to uniquely identify an object. You may also see the term GUID (globally unique identifier) which is also used. Both UUIDs and GUIDs are generated using random numbers, which makes it extremely unlikely that two of them will be the same.
+</p>
+<br>
+<h4>What version UUID generator is this tool?</h4>
+<p>This UUID generator generates a version 4 universally unique identifier. A version 4 UUID is made up of 32 hexadecimal digits (128 bits), divided into five groups separated by hyphens in this format: Xxxxxxxx-xxxx-4xxx-xxxx-xxxxxxxxxxxx.
```

Comment:
> 32 hexadecimal digits 

Suggestion:

> 32 hexadecimal values

A digit implies `0-9` and hex is `0-F`.

Reply:
Updated! (responding from vscode so I'm seeing how the comment responses look in github)